internal/backends/winit/Cargo.toml

```diff
@@ -73,7 +73,8 @@ accesskit_winit = { version = "0.23", optional = true }
 copypasta = { version = "0.10", default-features = false }
 
 [target.'cfg(not(any(target_family = "windows", target_os = "macos", target_os = "ios", target_arch = "wasm32")))'.dependencies]
-ashpd = { version = "0.9.2" }
+# Use same version and executor as accesskit
+zbus = { version = "4.4.0", default-features = false, features = ["async-io"] }
```

Comment:
The thing is that zbus is a private dependency of accesskit, and they may upgrade zbus in a minor version. which would then still result in two version of zbus :-/

Reply:
Yes. That's a rather common problem we have, right? The same problem exists with the ashpd dependency and many others.

I can remove the comment, if you think that it's redundant.

Reply:
up to you.
We can't use zbus 5 yet because of our MSRV anyway.
But we want to change our MSRV soon probably